src/js/client/components/RecipeForm.js

```diff
@@ -444,6 +451,34 @@ export default class RecipeForm extends React.Component {
     recipeInstruction.focus();
   }
 
+  handleValidator(object) {
+    let isValid = false;
+    for (var property in object) {
+      if (object.hasOwnProperty(property)) {
+        if (object[property] === '' || !validator.isURL(object['imageUrl'])) {
+          return this.setState({
+            error: `All Fields Required`
```

Comment:
Might be worth creating further down the line an "error" component that has an array of errors that we can pick from and reuse everywhere in the app.
i.e
```
errorMsgs = [ 
"Please put in a valid URL",
"Invalid Login. Please check your email or password is correct",
"Username already taken",
"Invalid quantity, please enter a valid number"
]
```
don't have to do this now but just wanted to bring it into the light